pkgs/development/python-modules/highctidh/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  pytestCheckHook,
+  fetchgit,
+}:
+buildPythonPackage rec {
+  pname = "highctidh";
```

Comment:
Please change the commit and PR names to `python3Packages.highctidh`, else Ofborg [won't be able to find them](https://github.com/NixOS/nixpkgs/pull/402850/checks?check_run_id=41362531529).